src/main.js

```diff
@@ -55,6 +59,25 @@ const filterBeforeCreate = (toast, toasts) => {
 
 const app = createApp(App).use(store).use(router);
 
+if (
+  ["staging", "production"].includes(process.env.NODE_ENV) &&
+  process.env.VUE_APP_SENTRY_DSN
+) {
+  // Since Vue3 isn't officially supported yet by Sentry, we're using
+  // the JavaScript Sentry integration with @sentry/browser.
+  Sentry.init({
+    dsn: process.env.VUE_APP_SENTRY_DSN,
+    integrations: [new Integrations.BrowserTracing()],
+    // Set tracesSampleRate to 1.0 to capture 100%
+    // of transactions for performance monitoring.
+    tracesSampleRate: 1.0,
```

Comment:
Can you provide a doc link for this, for further reading?

Reply:
Done.

Reply:
I've added a comment within the code to refer more configs.